tests/test_wsgi_servers.py

```diff
@@ -119,14 +140,17 @@ def _waitress_args(host, port):
     )
 
 
-@pytest.fixture(params=['gunicorn', 'meinheld', 'uvicorn', 'uwsgi', 'waitress'])
+@pytest.fixture(
+    params=['cheroot', 'gunicorn', 'meinheld', 'uvicorn', 'uwsgi', 'waitress']
+)
 def wsgi_server(request):
     return request.param
```

Comment:
```suggestion
def wsgi_server(request):
    wsgi_server_importable_name = request.param
    pytest.importorskip(wsgi_server_importable_name)
    return wsgi_server_importable_name
```

Reply:
Thanks, I linked to your snippet from #2480!
We wrangle some servers like Gunicorn+Meinheld in a bit more involved way (since the combo needs both dependencies), but it shouldn't be too hard to extend the concept.